content/service_dialogs/transform-vm.yml

```diff
@@ -18,6 +18,82 @@
       display_method_options:
       position: 0
       dialog_fields:
+      - name: tag_category
+        description: Tag category.
+        data_type: string
+        notes:
+        notes_display:
+        display: edit
+        display_method:
+        display_method_options: {}
+        required: false
+        required_method:
```

Comment:
I'm not sure what is the convention - since in other yaml files, if the field isn't required, the 'required_method:' element isn't specified at all.

@gmcculloug Could you advise ?

Reply:
The service dialogs are exported based on database tables so it is best to retain the full export format.  The other "provisioning" dialogs are free-form and get stored in a serialized column.